index.js

```diff
@@ -1,88 +1,25 @@
-'use strict';
+const findCentroid = require('./lib/find-centroid');
+const transCoord = require('./lib/trans-coord');
+const invTransCoord = require('./lib/inv-trans-coord');
+const sortByOrdinate = require('./lib/sort-by-ordinate');
+const haversine = require('./lib/sines').haversine;
```

Comment:
puedes usar el desconstructor sintaxis:

`js
const {haversine, ahaversine}= require('./lib/sines')
`

Reply:
good to know!